### PR TITLE
infra: remove version check for groovy .travis.yml

### PR DESCRIPTION
**Pull request by romani**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ before_script:
         export JAVA_HOME=/usr/lib/jvm/adoptopenjdk-11-hotspot-ppc64el
         export PATH=$JAVA_HOME/bin:$PATH
     fi
-  - groovy --version
+  # - groovy --version
```
Comment (Vyom-Yadav) [inline .travis.yml:44]: The problem is with `JAVA_HOME` not being correctly set up.

**Comment by romani** [comment]
Yes, I just tested an easy way to disable it or make it always green.

**Comment by Vyom-Yadav** [comment]
> IMO we need to stop using Travis completely, it is not worth the effort to maintain. We have plenty of reliable CI.

Do we want to migrate to existing ones like GA, CircleCI, etc? If we are looking for new ones, then GitLab CI can be a viable option. They got plans for open source https://about.gitlab.com/solutions/open-source/join/

**Comment by romani** [comment]
I doubt they can allow use them just as CI, they need hosting of sources.

**Comment by Vyom-Yadav** [comment]
> I doubt they can allow use them just as CI, they need hosting of sources.

There is a dedicated option to use GitLab CI for externally hosted projects which automatically creates a mirror from GitHub.

**Comment by rnveach** [comment]
> for externally hosted projects which automatically creates a mirror from GitHub

But this will not work for PRs, right?

**Comment by Vyom-Yadav** [comment]
I will create a local copy of Checkstyle to demonstrate and check it.

**Pull request closed without merging**